packages/backend/src/database/entities/savedCharts.ts

```diff
@@ -10,11 +10,12 @@ type DbSavedChart = {
     space_id: number;
     name: string;
     created_at: Date;
+    description?: string | undefined;
```

Comment:
is undefined redundant with `description?` (I never know how to use it)

Reply:
“?” operator means that parameter is optional, it's not necessary to pass it as a prop, but undefined means it should be passed in, but its value can be undefined.

Good call btw, the "?" was something i added on a later commit and i forgot to clean up after it.